storybookjs/storybook
# A11y design enhancements

## What I did

I have modified the X and check marks to be more accessible because the color of the X alone is difficult extract meaning from if you are visually impaired. They now are colored and list out the severity/urgency of the issue (which really provides better information anyway).
OLD:
![image](https://user-images.githubusercontent.com/29763590/56376240-4e3c1700-61d5-11e9-9e09-d018c54078f4.png)

NEW:
![image](https://user-images.githubusercontent.com/29763590/56376773-cc4ced80-61d6-11e9-9981-3ece97cba70a.png)

I also updated the text that appears when there aren't any rules within a tab. It used to use the word "A11y" and now it uses the word "accessibility". This helps people that don't understand what a11y means.
OLD:
![image](https://user-images.githubusercontent.com/29763590/56376332-8a6f7780-61d5-11e9-80e2-8a4d27262acb.png)

NEW:
![image](https://user-images.githubusercontent.com/29763590/56376369-a07d3800-61d5-11e9-8fe2-5b962702296c.png)

The global checkbox and all of the badges now respond nicely (when the addons panel is mobile, vertical, or horizontal).
OLD:
![image](https://user-images.githubusercontent.com/29763590/56376461-f05bff00-61d5-11e9-9741-66899ea80e0f.png)

NEW:
![image](https://user-images.githubusercontent.com/29763590/56376888-159d3d00-61d7-11e9-93fb-df2022f81663.png)

Capitalized the first character in colorblindness emulation text
![image](https://user-images.githubusercontent.com/29763590/56571848-d2065280-658b-11e9-974b-6e50c931de2b.png)


## How to test

- Is this testable with Jest or Chromatic screenshots? Sure
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.
